packages/react/src/components/InAppMessaging/Backdrop/Backdrop.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { Flex } from '../../../primitives/Flex';
+
+import { BackdropProps } from './types';
+
+export function Backdrop({ children, ...rest }: BackdropProps): JSX.Element {
+  return (
+    <Flex className="amplify-inappmessaging-backdrop" {...rest}>
```

Comment:
We can discuss offline and update before release if we need to, but thinking that maybe it would be cool not to have the feature name be a part of the `className`